cloud/backup-restore-cloud.md

```diff
@@ -9,25 +9,24 @@ tags: [recovery, failures]
 # Backup and restore
 
 Timescale Cloud automatically handles backup and restore for all
-services. To manage this, we use pgBackRest. There is no need to perform
+services using the `pgBackRest` tool. You don't need to perform
 backups for your Timescale Cloud service manually.
 
 Timescale Cloud automatically creates one full backup every week, and
-incremental backups every day. Additionally, all WAL ([Write-Ahead Log]
-[wal]) files are retained back to the oldest full backup. This means
+incremental backups every day. Additionally, all WAL
+([Write-Ahead Log][wal]) files are retained back to the oldest full backup. This means
 that you always have a full backup available for the current and
 previous week, and your service can be restored to any point during this
 time period.
 
-In the event of a storage failure, a service will automatically restore
+In the event of a storage failure, a service automatically restores
 from backup to the point of failure. In the event of a user error where
-a point-in-time recovery needs to be done, please contact support. We do
-not currently offer point-in-time recovery through the UI.
+a point-in-time recovery needs to be done, please [contact support][support]. 
 
-To understand how a restore from backup occurs after a storage failure,
-please see the following diagram. (Note: compute failures are able to
+This diagram helps you to understand how a restore from backup occurs after a storage failure.
+<Highlight type="note">compute failures are able to
 avoid a full restore from backup. For more information, see our docs on
```

Comment:
```suggestion
avoid a full restore from backup. For more information, see the
```

Reply:
I'm not sure I understand what this means?

Reply:
I'll rework this -- thanks!